### PR TITLE
Code split

### PR DESCRIPTION
**Pull request by lynnerang**
Splits codebase between current version of Specter and new 2.0 version

### CODE REVIEW

File: specter2/src/views/Annotations/Annotation/Body.svelte [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+    item.custom = item.custom.filter((itemField) => {
+      const isMatch = Object.keys(field).every(key => field[key] === itemField[key]);
+      return !isMatch;
+    });
+  };
+
+  // watch for updates to non-UI-specific fields and send update of nodeData
+  const uiFields = { position: null, selected: null };
+
+  $: if (JSON.stringify({ ...item, ...uiFields }) !== JSON.stringify({ ...savedItem, ...uiFields })) {
```
Comment (gblakeman) [inline specter2/src/views/Annotations/Annotation/Body.svelte:164]: Add a comment to let us know what this is diffing?

File: specter2/src/views/Home/SpecSelector.svelte [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+            bind:value={newSpecName}
+          />
+        </span>
+        {/if}
+    </div>
+    <div class="options-container">
+      <span class="header">Options</span>
+      <div class="content">
+        {#if newPage}
+        <span class="form-row setting" class:divider={!newPage}>
```
Comment (gblakeman) [inline specter2/src/views/Home/SpecSelector.svelte:177]: Indent this block after the `#if`

File: specter2/src/views/NavBar.svelte [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+    100% { opacity: 1; }
+  }
+</style>
+
+{#if currentView}
+<div class={`footer-bar ${currentView}`}>
+  <nav class="nav">
+    <ul>
+      {#each menuItems as item}
+      <li>
```
Comment (gblakeman) [inline specter2/src/views/NavBar.svelte:167]: Indent this block after `#each`

File: specter2/src/views/NavBar.svelte [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+  @keyframes flash {
+    0% { opacity: 1; }
+    50% { opacity: 0.1; }
+    70% { opacity: 1; }
+    100% { opacity: 1; }
+  }
+</style>
+
+{#if currentView}
+<div class={`footer-bar ${currentView}`}>
```
Comment (gblakeman) [inline specter2/src/views/NavBar.svelte:163]: Indent this block after `#if`

File: specter2/src/views/NavBar.svelte [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+      {/if}
+      </button>
+      <button
+        class="nav-item"
+        disabled={!selectedLayers.length}
+        aria-label="Add bounding box"
+        on:click={() => dispatch('handleAction', { action: 'bounding', payload: { currentView } })}
+      >
+        {#if selectedLayers.length && allSelectionsBounding}
+        <div class="flashit">
```
Comment (gblakeman) [inline specter2/src/views/NavBar.svelte:218]: Indents

**Review by gblakeman — APPROVED** [review]
🎉👊

Gave this a quick review. Flagging a bunch of little things…

File: specter2/src/views/NavBar.svelte (hunk 5 above)
Comment (lynnerang) [reply specter2/src/views/NavBar.svelte:218]: Thanks!  Resolved in spacing branch

File: specter2/src/views/NavBar.svelte (hunk 3 above)
Comment (lynnerang) [reply specter2/src/views/NavBar.svelte:167]: Thanks!  Resolved in spacing branch

File: specter2/src/views/NavBar.svelte (hunk 4 above)
Comment (lynnerang) [reply specter2/src/views/NavBar.svelte:163]: Thanks!  Resolved in spacing branch

File: specter2/src/views/Home/SpecSelector.svelte (hunk 2 above)
Comment (lynnerang) [reply specter2/src/views/Home/SpecSelector.svelte:177]: Thanks!  Resolved in spacing branch

File: specter2/src/views/Annotations/OrientationSelector.svelte [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+>
+  <div class="content-wrapper">
+    {#if selectedItemCount}
+      <p class="instructions">
+        Select an annotation orientation to set for <strong>{selectedItemCount}</strong> selected <span class={currentView}><strong>{currentView}</strong></span> design layer{selectedItemCount > 1 ? 's' : ''}
+      </p>
+      <div class="orientation-wrapper orientation">
+        <div class="orientation-radios">
+          {#each orientationOptions as opt}
+          <div class={`orientation-radio ${opt}`}>
```
[the remark this answers is not in the document]
Comment (lynnerang) [reply specter2/src/views/Annotations/OrientationSelector.svelte:168]: Thanks!  Resolved in spacing branch

**Pull request closed without merging**